test/prisma.test.ts

```diff
@@ -0,0 +1,13 @@
+import { cwd } from "node:process";
+import { integrationTest, readAppmap, runAppmapNode } from "./helpers";
+import { copyFileSync, rmSync } from "node:fs";
+import { spawnSync } from "node:child_process";
+
+integrationTest("mapping Prisma tests", () => {
+  const testDir = `${cwd()}/test/prisma`;
```

Comment:
That's not very robust — what if jest is run from a different directory? You can use `resolveTarget()` to get it.